packages/perseus/src/widgets/radio/__tests__/radio.testdata.ts

```diff
@@ -323,3 +323,96 @@ export const multiChoiceQuestionAndAnswer: [
         [0, 1, 2, 3],
     ],
 ];
+
+export const shuffledQuestion: PerseusRenderer = {
```

Comment:
**thought**: The `interactive-graph` team built a whole test data generator api using the builder pattern. I long for a day when we have builders for all types of widgets. 

```ts
const shuffledQuestion = new RendererBuilder()
    .withRadioWidget()
        .withIncorrectAnswer("Incorrect Choice 1")
        .withIncorrectAnswer("Incorrect Choice 2")
        .withCorrectAnswer("Correct Choice")
        .withIncorrectAnswer("Incorrect Choice 3")
        .randomized()
        .endWidget()
    .build()
```

No action required.